service/resource/certificate/certificate.go

```diff
@@ -17,7 +19,8 @@ type Config struct {
 	K8sClient kubernetes.Interface
 	Logger    micrologger.Logger
 
-	CertificateDirectory string
+	CertificateDirectory  string
+	CertificatePermission int
```

Comment:
We can configure the permission we write certificates for.

Reply:
Why does the type differ between config and resource?

Reply:
Because I'm dumb. Fixed.